packages/zwave-js/src/lib/driver/ZWaveOptions.ts

```diff
@@ -59,6 +59,9 @@ export interface ZWaveOptions {
 	 */
 	skipInterview?: boolean;
 
+	/** Allows you to skip user code interview */
+	skipUserCodeInterview?: boolean;
```

Comment:
if we want to flip the default behavior to skip by default, we can maybe rename this `doNotSkipUserCodeInterview` or `interviewUserCodes`.

Reply:
I'd make **not scanning** user codes the default for 8.0.0 and call the flag to restore the previous behavior `queryAllUserCodes`.
Moving `skipInterview` there is okay, we just need to update tests that use it.